scripts/examples/python/setuppy_gen.ml

```diff
@@ -0,0 +1,11 @@
+let filename = Filename.concat "." "setup.py"
+
+let setup_templ = Mustache.of_string
+    "from setuptools import setup\n\
+     \n\
+     setup(version='{{version}}')\n"
+
+let json = `O [ "version", `String Datamodel_common.api_version_string ]
+
+let () =
+  Xapi_stdext_unix.Unixext.write_string_to_file filename @@ Mustache.render setup_templ json
```

Comment:
I think in this particular case mustache might be a bit of an overkill since the rendered file is so small that a simple print would suffice - unless you're planning with extensibility in mind.

Reply:
I can change it. I used it because it was already being used elsewhere and what takes the most time to compile is `Datamodel_common` in any case.